src/i18n/locales/en/translation.json

```diff
@@ -594,6 +594,12 @@
         "pay_now": "Pay now"
       }
     },
+    "confirm_request_permission": {
+      "title": "Approve Request",
+      "allow": "Allow {{host}} to:",
+      "enable_method": "Send request {{method}}",
```

Comment:
Is `Send request` the right wording here? @bumi 
Maybe `Call` is better? I don't like the word send` in there because sending in this context is associated with sending money. (`send request `sendPayment`)

Reply:
Changed string to "call". 
And removed `{{method}}` from the "always allow" string as there is no string mapping yet. 
A mapping can be only relevant when this follow-up ticket is addressed: https://github.com/getAlby/lightning-browser-extension/issues/1725

Screenshot is updated too.